superchain/configs/sepolia/superchain.yaml

```
@@ -5,5 +5,6 @@ l1:
   explorer: https://sepolia.etherscan.io
 
 protocol_versions_addr: "0x79ADD5713B383DAa0a138d3C4780C7A1804a8090"
+superchain_config_addr: "0xc2be75506d5724086deb7245bd260cc9753911be"
```

```suggestion
superchain_config_addr: "0xC2Be75506d5724086DEB7245bd260Cc9753911Be"
```

Nit: use the checksum formatting